src/more-infos/more-info-climate.html

```diff
@@ -309,7 +325,7 @@
     this.fanIndex = -1;
     if (this.stateObj.attributes.fan_list) {
       this.fanIndex =
-          this.stateObj.attributes.fan_list.indexOf(this.stateObj.attributes.fan_list);
+          this.stateObj.attributes.fan_list.indexOf(this.stateObj.attributes.fan_mode);
```

Comment:
This was a bug - Should I raise an issue and link it in the PR description as well?

Reply:
Raised #852 to make sure it's tracked.